packages/web3/networks.ts

```diff
@@ -0,0 +1,65 @@
+import envParsed from "./envParsed";
+
+const TESTNET_PRIVATE_KEY = envParsed().TESTNET_PRIVATE_KEY;
+const MAINNET_PRIVATE_KEY = envParsed().MAINNET_PRIVATE_KEY;
+const API_KEY = envParsed().ETHERSCAN_API_KEY;
+
+const sepolia = {
```

Comment:
Is this network ETH Sepolia? In that case the chainId and urls should be changed

Reply:
done!